py/DLA.py

```diff
@@ -85,9 +85,9 @@ def get_N(z, Nmin=19.5, Nmax=22.0, nsamp=100):
     nn = np.linspace(Nmin,Nmax,nsamp)
     probs = np.zeros([Nz,nsamp])
     if use_pyigm:
-        auxfN = np.cumsum(fN_default.evaluate(nn,z), axis=0).T
-        probs_low = auxfN[:,1:]
-        probs_high = auxfN[:,:-1]
+        auxfN = fN_default.evaluate(nn,z) 
+        # Above we got logprob in a grid NHI,z (note the order, we will transpose) 
+        probs = (10**auxfN/np.sum(10**auxfN, axis=0)).T # The probability is the function divided by the sum for a given z
```

Comment:
The old code made variables "probs_low" and "probs_high", which were then combined to make "probs" in line 94, whereas now "probs" is generated in line 90 and line 94 returns an error as "probs_low" and "probs_high" are undefined.

Reply:
@fjaviersanchez I don't know how you'd prefer this to work, it's up to you really!

Reply:
I thought @fjaviersanchez had already tested the code and made some plots... Isn't this the case, Javi?

Reply:
Yes, sorry. I tested the code but I forgot add the commit with last change (putting the probs_high-probs_low part inside of the if clause... 😅

Reply:
Great thanks for sorting that!